docs/docs/general/06-Tutorials/setup-android.md

```diff
@@ -0,0 +1,128 @@
+# Setting up Android Devices for Testing Mobile Apps
+
+In this tutorial, we will walk you through how to set up an Android device for developing and testing mobile apps running on Android devices. Depending on the testing environment, i.e. whether it is a real Android device or an emulator, the procedure might slightly differ. But the overall process of the setup will be more or less the same. 
+
+### Requirements
+- [Android Studio](https://developer.android.com/studio) installed
```

Comment:
You need only the adb and can install it manually. See

`Or if you want the standalone Android SDK Platform-Tools package, you can [download it here](https://developer.android.com/studio/releases/platform-tools).` https://developer.android.com/studio/command-line/adb

Reply:
@programminx-askui @JohannesDienst-askui  Thank you for the review. I have added a new section that explains how to set up without the android studio. 

https://github.com/HaramChoi-askui/askui/blob/90d3b03dbb84d0bbc0396827ea80b48225dc7d2d/docs/docs/general/06-Tutorials/setup-android.md?plain=1#L83-L141